Vagrantfile

```diff
@@ -44,71 +19,76 @@ raise "Configuration #{CONFIGURATION} missing from config.yml" unless config[CON
 PREFIX = ''
 
 # Generates a hosts file
-if (INSTALLATION == 'salt') then
-  hosts = Vagrant::Hosts.new(config[CONFIGURATION]['nodes'], 
-                             selected = 'public', domain='ceph', 
-                             aliases={ 'admin' => 'salt' })
-elsif (INSTALLATION == 'openattic') then
-  hosts = Vagrant::Hosts.new(config[CONFIGURATION]['nodes'], 
-                             selected = 'public', domain='ceph')
+# if (INSTALLATION == 'salt') then
+#   hosts = Vagrant::Hosts.new(config[CONFIGURATION]['nodes'],
+#                              selected = 'public', domain='ceph',
+#                              aliases={ 'admin' => 'salt' })
+# elsif (INSTALLATION == 'openattic') then
+#   hosts = Vagrant::Hosts.new(config[CONFIGURATION]['nodes'],
+#                              selected = 'public', domain='ceph')
+if File.exists?("#{ENV['VC_CONFIG']}")
```

Comment:
should we also have "VC_CONFIG=config/sles15.yml " some of these files by default ? can you show one as example ?